### PR TITLE
feat(ui): Add a help message describing how to see help messages

### PR DESCRIPTION
**Pull request by ziproot**
**Feature**

This PR addresses the feature described in discussion #10776

## Summary
While the game shows help messages, they are shown once, and it can be difficult to figure out that it is possible to see them again. This PR adds a third “basic” help message which describes how to view help messages or reactivate them, so that players know that they can review these messages as necessary.

## Screenshots
image
EDIT: Updated screenshot

## Testing Done
Made sure the help messaged showed up for the first few days, then did not show up again even after reactivating help.

## Save File
Make a new pilot.

## Performance Impact
Not zero, but barely so.

### CODE REVIEW

**Review by TheGiraffe3 — APPROVED** [review]

**Review by TomGoodIdea — APPROVED** [review]
I'm not sure if this additional message will really improve our current situation with the help system, but here's my code approval.

File: data/_ui/help.txt [hunk 1]
```diff
@@ -25,6 +25,9 @@ help "basics 1"
 help "basics 2"
 	`For the main menu, press <Show main menu>. The control key bindings can be viewed and changed in the preferences.`
 
+help "basics 3"
+	`To see a help message again, press <Show help>. To see all help messages again, go to the main menu, go to preferences, go to settings, go to page two, then select "Reactivate first-time help."`
```
Comment (roadrunner56) [inline data/_ui/help.txt:29]: Trying to reduce repetitiveness here.
Any reason why <Show help> is in brackets and "Reactivate first-time help" is in quotes?
```suggestion
	`To see a help message again, press <Show help>. To see all help messages again, go to the main menu, select Preferences, then go to Settings. Navigate to page two, then select "Reactivate first-time help."`
```

Comment (TomGoodIdea) [reply data/_ui/help.txt:29]: It replaces Show help with your current key assigned to it (usually F1).

**Review by roadrunner56 — APPROVED** [review]
Grammar looks good.

**Comment by xobes** [comment]
😁 Do we want to add a fourth basic help to tell the player they can hit "/" to review all the messages that spew across the screen?

/me ducks

**Comment by ziproot** [comment]
> Given that it's fairly common for people to either miss, ignore, or forget about the help messages they are already shown, do you actually expect this to be the one thing that isn't ignored or forgotten or is this just to make us feel better give us another thing to point to: "You were told about it in this message upon creating a new pilot."

 I was responding to this:
 
 > OK, except there is no hint of there being a help menu, and F1 doesn't work in the main screen (planet view), so it isn't really obvious to find. :-(
https://github.com/endless-sky/endless-sky/discussions/10776#discussioncomment-11229254

**Comment by warp-core** [comment]
> > Given that it's fairly common for people to either miss, ignore, or forget about the help messages they are already shown, do you actually expect this to be the one thing that isn't ignored or forgotten or is this just to make us feel better give us another thing to point to: "You were told about it in this message upon creating a new pilot."
> 
> I was responding to this:
> 
> > OK, except there is no hint of there being a help menu, and F1 doesn't work in the main screen (planet view), so it isn't really obvious to find. :-(
> > [#10776 (reply in thread)](https://github.com/endless-sky/endless-sky/discussions/10776#discussioncomment-11229254)

Yes. That being an example of a person missing, ignoring, or forgetting about a help pop up they were already shown. My question is, is there an expectation that this help message about help messages will be remembered (unlike, apparently, other help pop ups) or is this just a thing we're going to and tell people they should pay more attention when they ask about things that are described in the existing help pop ups?

**Comment by ziproot** [comment]
remove waiting on dev?

**Pull request merged**